miranda/convert/data/ecmwf_cf_attrs.json

```diff
@@ -179,6 +189,11 @@
         "era5-single-levels": "J m-2",
         "era5-single-levels-preliminary-back-extension": "J m-2"
       },
+      "_invert_sign": {
+        "era5-land": true,
+        "era5-single-levels": true,
+        "era5-single-levels-preliminary-back-extension": true
```

Comment:
I believe it is the opposite!

ERA5 specifies : "The ECMWF convention for vertical fluxes is positive downwards."
And CF conventions for "surface_downwelling_shortwave_flux_in_air", which is listed as `rsds` says : "The sign convention is that "upwelling" is positive upwards and "downwelling" is positive downwards."